.github/workflows/tests.yml

```diff
@@ -0,0 +1,38 @@
+name: Camayoc Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: ["3.6", "3.9"]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        make install-dev
+        pip install codecov
+    - name: Lint
+      run: |
+        make lint
+    - name: Tests
```

Comment:
Same with this name. It's not really running any tests, but rather creating the code cov report with pytest, and also generating/validating the documentation.

If it still works (I'm not sure if the documentation needs a specific order), we might want to think about moving the `make test-coverage` line in the `Code Coverage` Section, and rename this one to `Generate Documentation` or something...

Reply:
This step is actually running the tests. When pytest is called with --cov* flags it is also running the tests and measuring their coverage (evidence of this behavior [in the output](https://github.com/quipucords/camayoc/runs/4705583893?check_suite_focus=true#step:6:19))

Maybe this step could be renamed to "Run tests and collect coverage".

Reply:
It's running the unit tests for camayoc, not qpc though

Reply:
Oh, ok, I see your point now